sw/tools/airframe_editor/xml_airframe.py

```diff
@@ -76,13 +92,18 @@ def reorganize_airframe_xml(airframe_xml):
     find_and_add_sections_with_name(airframe_xml, airframe, "XSENS")
 
     find_or_add_group(airframe_xml, airframe, "GAINS")
+
     # Fixedwing
-    find_and_add_sections_with_name(airframe_xml, airframe, "HORIZONTAL CONTROL")
+    find_and_add_sections_with_name(airframe_xml, airframe,
```

Comment:
One of my student started a new design (in Python) for the Paparazzi Center two years ago. You can have a look at this page: http://wiki.paparazziuav.org/wiki/Paparazzi_Center/Evolutions
The code is here now: https://github.com/paparazzi/paparazzi/tree/master/sw/supervision/python
Several things could be done, starting by fixing the paths (it doesn't work if you don't start the program from its own directory). I also wanted to improve the design a bit, by hiding some panels when they are not needed/wanted for instance.